### PR TITLE
[AIRFLOW-211] Fix JIRA "resolve" vs "close" behavior

### PR DESCRIPTION
**Pull request by jlowin**
Dear Airflow Maintainers,

Please accept this PR that addresses the following issues:
- https://issues.apache.org/jira/browse/AIRFLOW-211

### CODE REVIEW

File: dev/airflow-pr [hunk 1]
```diff
@@ -443,17 +445,6 @@ def resolve_jira_issue(comment=None, jira_id=None, merge_branches=None):
         raise ValueError(
             "ASF JIRA could not find issue {}\n{}".format(jira_id, e))
 
-    if comment is None:
```
Comment (jlowin) [inline dev/airflow-pr:446]: Note: these comment lines are more appropriate after the "Proceed with Airflow-211?" question, so this just moves them down.

**Comment by criccomini** [comment]
LGTM but the commit msg needs fixing. :)

**Comment by jlowin** [comment]
@criccomini yes indeed it does :)

**Comment by criccomini** [comment]
wait 207 or 211?

**Comment by jlowin** [comment]
Shoot. Trying to do too many things at once. I'm glad you're more on top of this than I am...

**Comment by criccomini** [comment]
LGTM

**Pull request closed without merging**